modules/weblog/testdata/u_ex221107.log

```diff
@@ -0,0 +1,168 @@
+#Software: Microsoft Internet Information Services 10.0
+#Version: 1.0
+#Date: 2022-11-07 14:29:06
+#Fields: date time s-ip cs-method cs-uri-stem cs-uri-query s-port cs-username c-ip cs(User-Agent) cs(Referer) sc-status sc-substatus sc-win32-status time-taken
+2022-11-07 14:29:06 127.0.0.1 GET /us - 80 - 127.0.0.1 Go-http-client/1.1 - 404 0 2 130

```

Comment:
404 0 2 130
$status - - $request_time

Is request time in microseconds?

Reply:
It is [miliseconds](https://learn.microsoft.com/en-us/windows/win32/http/w3c-logging)

Reply:
it is a problem then https://github.com/netdata/go.d.plugin/blob/ba33f91486c6372d6b0a7bd81a7edcb1b4d6f844/modules/weblog/logline.go#L609-L616